cmd/report.go

```diff
@@ -346,3 +353,19 @@ func saveReport(baseName string, ext string, nodeFilter string, content string)
 	fmt.Printf("%s report saved to %s\n", strings.Title(baseName), reportPath)
 	return nil
 }
+
+func toAnonyimize() bool {
+
+	c, error := reporting.NewDefault()
```

Comment:
I see that we are copying the `NewDefault()` function name from `credentials.NewDefault()` but the name seems strange to me, because it isn't just the default config values. I don't think you need to change it here since it also requires a change in go-libs, but what would you think about `ReadConfig()` or `LoadedConfig()` or something like that in the future?

Reply:
Yes, this name is likely inspired from go-libs. It was a bit confusing for me as well. I can add this item in the tech-debt / improvement list.